frontend/src/components/pie.rechart.tsx

```diff
@@ -48,6 +57,4 @@ class PieRechartComponent extends React.Component<PieData> {
       </>
     )
   }
-}
-
-export default PieRechartComponent 
```

Comment:
Exporting defaults is a bad habit and should only be used for modules. You should instead export the individual elements to ensure consistency with naming conventions